File: exercises/pset6/cash/cash.py
from cs50 import get_float
while True:
    change = get_float("Change owed:")
    if change > 0:
        break
change *= 100
a = int(change/25)
count = a
change = change - a * 25
b = int(change/10)
count += b
change = change - b * 10
c = int(change/5)
count += c
change = change - c * 5
d = int(change/1)
count += d
change = change - d
print(f"{count}")

File: exercises/pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h> //Biblioteca to CS50
#include <math.h>
int main(void)
{
    float i = 0;
    do
    {
        i = get_float("Troco devido: ");
    }
    while (i <= 0);
    int v = round(i * 100);
    int g = 0;
    if (v >= 25)
    {
        do
        {
            v = v - 25;
            g ++;
        }
        while (v >= 25);
    }
    else;
    if (v >= 10)
    {
        do
        {
            v = v - 10;
            g ++;
        }
        while (v >= 10);
    }
    else;
    if (v >= 5)
    {
        do
        {
            v = v - 5;
            g ++;
        }
        while (v >= 5);
    }
    else;
    if (v >= 1)
    {
        do
        {
            v = v - 1;
            g ++;
        }
        while (v >= 1);
    }
    else;
    printf("Total de moedas: %i\n", g);
}